docs/integrations/sources/zoho-crm.md

```diff
@@ -0,0 +1,137 @@
+| AU          | https://zohoapis.com.au |
+| EU          | https://zohoapis.eu     |
+| IN          | https://zohoapis.in     |
+| CN          | https://zohoapis.com.cn |
+| JP          | https://zohoapis.jp     |
+
+### Sandbox
+
+| Environment | Endpoint                        |
+|:------------|:--------------------------------|
+| US          | https://sandbox.zohoapis.com    |
+| AU          | https://sandbox.zohoapis.com.au |
+| EU          | https://sandbox.zohoapis.eu     |
+| IN          | https://sandbox.zohoapis.in     |
+| CN          | https://sandbox.zohoapis.com.cn |
+| JP          | https://sandbox.zohoapis.jp     |
+
+### Developer
+
+| Environment | Endpoint                           |
+|:------------|:-----------------------------------|
+| US          | https://developer.zohoapis.com     |
+| AU          | https://developer.zohoapis.com.au  |
+| EU          | https://developer.zohoapis.eu      |
+| IN          | https://developer.zohoapis.in      |
+| CN          | https://developer.zohoapis.com.cn  |
+| JP          | https://developer.zohoapis.jp      |
+
+For more information about available environments, please visit [this page](https://www.zoho.com/crm/developer/sandbox.html?src=dev-hub)
+
+### Performance considerations
+
+If you experience the long time for sync operation, please consider using smaller date range by tuning `start_date` parameter.
+Also, Zoho CRM API calls are associated with credits, each Zoho CRM edition has a limit in a 24-hour rolling window, so please, consider it when configuring your connections.
+More details about Zoho CRM API credit system can be found [here](https://www.zoho.com/crm/developer/docs/api/v2/api-limits.html).
+
+### Important Note
+
+Zoho Developer environment API is inconsistent with production environment API. In request to Metadata API it responds with about just a half a list of supported modules.
+This is a strongly misleading issue, so please take into account.
+
+## Getting started
+
+To set up a connection with a Zoho CRM source, you will need to choose start sync date, Zoho CRM edition, region and environment. The latest are described above. Except for those, you will need OAuth2.0 credentials - Client ID, Client Secret and Refresh Token.
+
+### Get Client ID, Client Secret, and Grant Token
+
+1. Log into https://api-console.zoho.com/
+2. Choose client
+3. Enter a scope the future refresh and access tokens will cover. For instance, it can be `ZohoCRM.modules.ALL, ZohoCRM.settings.ALL, ZohoCRM.settings.modules.ALL`. **Make sure the scope covers all needed modules**.
+4. Enter grant token's lifetime and description, click "Create".
+5. Copy Grant token, close the popup and copy Client ID and Client Secret on the "Client Secret" tab.
+
+### Create Refresh Token
+
+For generating the refresh token, please refer to [this page](https://www.zoho.com/crm/developer/docs/api/v2/access-refresh.html).
+Please, be aware that you are limited in time due to the grant token lifetime.
```

Comment:
```suggestion
Make sure to complete the auth flow quickly, as the initial token granted by Zoho CRM is only live for a few minutes before it can no longer be used to generate a refresh token. 
```

Reply:
fixed